Makefile

```diff
@@ -35,16 +35,16 @@ hooks: venv
 
 .PHONY: build
 build: venv  ## Compile and install Daft for development
```

Comment:
you also want to rename the target `venv` to `.venv` to enable makefile caching on the correct folder.

Reply:
👍 Make sense, updated.